### DISCUSSION TITLE
VSCode new install does nothing

### DISCUSSION THREAD

**Discussion by nmile** [community/community#58275] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I signed up for the trial, installed the extension, it did not prompt to connect to my github account. I don't believe I've ever authorized vscode with my github account, so I turned on settings sync, which did prompt to authorize my github account. Uninstalled, reinstalled the copilot extension. It says "this extension is enabled globally". Went through the getting started docs. Tried out the "seeing your first suggestions" - does nothing. Opened the Output, selected "Github copilot", outputs nothing. In the Configure Github Copilot docs, it says you can enable/disable the extension via the status icon in the bottom of vscode, I don't see that icon at all. When I click my profile button and click Github -> manage trusted extensions, it says "this account has not been used by any extensions"

I keep reading through the docs thinking I missed something. Not sure what else to try. I'm on Windows 11, VSCode 1.79.2, GitHub Copilot v1.89.156.

**Comment by the-officialjosh** [comment]
Not sure if you haven't tried any of these solutions but have a look at my own suggestions
Certainly! Let's focus on the steps that haven't been done yet. Please try the following:

1. **Check for updates**: Make sure both VS Code and the GitHub Copilot extension are up to date. Update them to the latest versions if needed.

2. **Verify extension status**: Look for the GitHub Copilot icon in the activity bar on the side of the VS Code window. It should appear as a small icon with a "Copilot" label. If you don't see it, try restarting VS Code after enabling the extension.

3. **Check the Output panel**: Open the Output panel in VS Code by going to View > Output (or using the shortcut Ctrl/Cmd + Shift + U), and select "GitHub Copilot" from the dropdown menu. This should display any relevant output or error messages from the extension.

4. **Reinstall the extension**: If none of the above steps works, you can try uninstalling the GitHub Copilot extension, **restarting** VS Code, and then reinstalling it again. This may help resolve any potential installation or configuration issues.

If the issue persists after following these steps, it might be helpful to consult the GitHub Copilot documentation or reach out to GitHub Support for further assistance, as they can provide more specific guidance based on your setup.

I hope this helps! Let me know if you have any other questions or concerns.

**Reply by nmile** [reply]
I've tried all these things and contacted support. Awaiting reply.

**Reply by the-officialjosh** [reply]
unistall Nightly copilot if you have it installed

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by nmile** [reply]
This just wanted me to sign back in to settings sync
![1](https://github.com/community/community/assets/8800932/db19507f-0655-4cde-853c-c4493611acd1)

**Comment by toruhitomi** [comment]
Hi,

I also had the same problem. After signing up for a free trial, I followed the instructions but didn't work.
Looking at the output of GitHub Copilot, it just says "inactive"
![image](https://github.com/community/community/assets/52917593/106c8cf6-f7ed-4e91-b19f-294d425d71d6)

I tried the steps described above including checking for updates, uninstalling and reinstalling extensions, signing out and signing back in but none let me start using it...
Any suggestions?

[VSCode Version info]
Version: 1.79.2
Commit: 695af097c7bd098fbf017ce3ac85e09bbc5dda06
Date: 2023-06-14T08:58:33.551Z
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.5.0 (Ventura 13.4, 22FF66)

[Copilot extension versions]

- GitHub Copilot: v1.89.156
- GitHub Copilot Nightly: v1.89.172
- GitHub Copilot Chat: v0.2.1

**Reply by nmile** [reply]
Still haven't found a solution, support told me to reinstall. Didn't work. You're getting more output than I am. Mine's just blank.

**Reply by rwedoff** [reply]
Try version Copilot Nightly version 1.82.16. I have had the same issue with the later version.

**Reply by toruhitomi** [reply]
Thanks for the comment @rwedoff!! 
It was because I overlooked the error message at the bottom right corner of VS Code, saying that we need to uninstall Github Copilot if you wanna use Copilot Nightly. Then, I was able to use the latest version of Nightly (v1.90.175) successfully :)

**Reply by nmile** [reply]
Thank you for the suggestion. I uninstalled the version I had, clicked the latest nightly from the marketplace. Didn't work. Uninstalled that, reloaded vsc, downloaded v1.89.167, installed that from vsix. Still just installs with no browser popup window authenticating it to github, still no icon at the bottom right of the screen, still no output from github copilot in the output window. But when I went into the plugin it said reload required. Clicked that and it automatically upgraded itself to the latest nightly build. Still nothing. Why isn't there a button to authenticate manually?

**Reply by toruhitomi** [reply]
When I had both Github Copilot and Nightly at the same time I had the following notification:
![image](https://github.com/community/community/assets/52917593/7314430e-1e08-4764-a52c-83b5548b04e5)

And I uninstall it by just clicking the 'Uninstall' button there (I didn't uninstall manually from Extension pane).
Then you'll be requested to reload and clicked 'Reload Required' button.
In my environment, this solved the problem.

Regarding the authentication, when you're already signed-in as your github account (you can check your status in the 'Accounts' icon at the bottom left corner of vsc), you will not see the pop-up.

**Reply by nmile** [reply, marked as answer]
Got it to work! I installed a VM with ubuntu just to have a fresh environment to try it on, everything worked as it should. So I uninstalled VSCode on windows, and realized it doesn't completely uninstall so I had to follow these instructions for a clean uninstall: https://code.visualstudio.com/docs/setup/uninstall, then I installed a fresh copy of VSCode, and went through the install process for copilot, and now it works. Something in my VSCode settings must have been messed up preventing that authentication popup, because that's what I was missing.

**Reply by Brent-Chiono** [reply]
So you didn't get it to work. Installing it on another machine / a VM is not a resolution. A resolution to a problem can not be "Use another computer".

Your answer should be: I could not get it to work, this is still unsolved, it works on other computers / VMs.
